aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/data/AWSCognitoAuthCredentialStore.kt

```diff
@@ -40,27 +38,9 @@ class AWSCognitoAuthCredentialStore(
     private var keyValue: KeyValueRepository =
```

Comment:
Can this be changed to lazy initialization so that it is only created when needed.
```
private val keyValue : KeyValueRepository by lazy { 
        keyValueRepoFactory.create(context, awsKeyValueStoreIdentifier, isPersistenceEnabled)
    }